### DISCUSSION TITLE
ExNihilo question/issue

### DISCUSSION THREAD

**Discussion by Bananaguns2** [NovaMachina-Mods/ExNihiloSequentia#383] [Q&A] [answered]
Is the loot table size for the sieves hard coded? I'm using latest version for 1.18.2 & some of my costum drops(datapack via 1.13+ feature implementation) are disappearing I'm assuming because there's a cap of how many items each block can drop when sieved?

**Comment by NovaMachina** [comment, marked as answer]
No, drops from sieves are not hard coded. They are standard json recipies. Please see the [wiki](https://wiki.jacob-williams.me/docs/) for more information.

**Reply by Bananaguns2** [reply]
Weird I'm using json format & some items are disappearing from the loot table in JEI(JER add-on for JEI)

**Reply by Bananaguns2** [reply]
It's all good I fixed it for some odd reason json script doesn't accept 1 .9 or .1 as chances whereas 1.0 0.9 & 0.1 worked fine
